### PR TITLE
Fix blank issue template URL

### PR DESCRIPTION
**Pull request by LilianBoulard**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@ contact_links:
     url: https://github.com/skrub-data/skrub/discussions
     about: Ask questions and discuss with other members of the community
   - name: Blank issue
-    url: https://github.com/scikit-learn/scikit-learn/issues/new
+    url: https://github.com/skrub-data/skrub/issues/new
```
Comment (LeoGrin) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
    url: https://github.com/python/cpython/issues/new
```

**Review by LeoGrin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (LilianBoulard) [reply .github/ISSUE_TEMPLATE/config.yml:6]: :joy:

Comment (GaelVaroquaux) [reply .github/ISSUE_TEMPLATE/config.yml:6]: :trollface:

**Review by GaelVaroquaux — APPROVED** [review]
LGTM. Merging

**Pull request merged**